src/components/Episodes/actions.js

```diff
@@ -0,0 +1,11 @@
+export const PREV_EPISODE 			= "PREV_EPISODE";
+export const NEXT_EPISODE 			= "NEXT_EPISODE";
+export const GET_EPISODE			= "GET_EPISODE";
+export const SHOW_TRAILER			= "SHOW_TRAILER";
+
+export default {
+	PREV_EPISODE,
+	NEXT_EPISODE,
+	GET_EPISODE,
+	SHOW_TRAILER
+};
```

Comment:
Wait, isn't `export default` going to cause a problem with the whole `import * as ACTIONS` thing in `index.js`? Or is that in the other branch? If it's in the other branch then you need to remember about this during the merge as some stuff might be overwritten - best to avoid refactoring the same file on multiple simultaneous branches or you're going to encounter issues like this one.  Unfortunately you can't go back and remove files from a commit, so it's just something you need to be mindful about.

Reply:
It's working fine like this. 
The ACTIONS import = an object containing the different actions.
Could still import them seperately if needed, and then use without the ACTIONS namespace (when importing into a component).